package.json

```diff
@@ -43,7 +43,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.7.7",
-    "cabal-client": "2.0.0",
+    "cabal-client": "^2.0.1",
```

Comment:
needs to update to fix this [bug](https://github.com/cabal-club/cabal-client/pull/21)
please `npm i` again.